x-pack/plugins/observability/public/pages/alerts/alerts_disclaimer.tsx

```diff
@@ -5,33 +5,55 @@
  * 2.0.
  */
 
-import React from 'react';
-import { EuiLink, EuiCallOut } from '@elastic/eui';
+import React, { useState, useEffect } from 'react';
+import { EuiLink, EuiCallOut, EuiButton, EuiSpacer } from '@elastic/eui';
 import { i18n } from '@kbn/i18n';
 import { FormattedMessage } from '@kbn/i18n/react';
 
+const LOCAL_STORAGE_KEY_MSG_ACK = 'xpack.observability.alert.ack.experimental.message';
+
 export function AlertsDisclaimer() {
+  const [experimentalMsgAck, setExperimentalMsgAck] = useState(false);
+
+  const dismissMessage = () => {
+    setExperimentalMsgAck(true);
+    localStorage.setItem(LOCAL_STORAGE_KEY_MSG_ACK, 'true');
+  };
+
+  useEffect(() => {
+    const isExperimentalMsgAck = localStorage.getItem(LOCAL_STORAGE_KEY_MSG_ACK);
+    setExperimentalMsgAck(isExperimentalMsgAck === 'true');
```

Comment:
@fkanout I know it is still a draft, but I spotted it should be boolean `setExperimentalMsgAck(isExperimentalMsgAck === true);`

Reply:
Nope, everything in localStorage is String unfortunately.
I recommended another solution but this is also ok.

Reply:
To avoid confusion, I used `JSON.parse` to challenge the value against boolean :
https://github.com/elastic/kibana/pull/117283/#discussion_r744444295